### PR TITLE
Create 贡献者许可协议常见问题.md

### PR DESCRIPTION
**Pull request by JesseZhou-1**
There are some legal-related terms that I am not sure if I translated correctly. if there is any mistranslated part, please kindly point out. Thanks!

### CODE REVIEW

**Comment by JesseZhou-1** [comment]
I also translate the "ASF许可证常见问题", but for some reason the new PR got under this one. So there is two two commits in this PR. 
In the translation of "ASF许可证常见问题", I am unfamiliar with a few terms in the"什么是 ASF 源代码的证明？" section, please have a close look on that part. Thanks!

**Comment by WillemJiang** [comment]
Hi @JesseZhou-1，thanks for your contribution. 
You need to create another branch and work on it if you have another PR to send.  

sorry of the delay of response. I will get back to you soon with some comments on the translation documentation.

**Comment by JesseZhou-1** [comment]
got it

**Review by WillemJiang — COMMENTED** [review]
这里有一些中英文空格的问题， 建议参考一下下面的要求
https://github.com/alc-beijing/translation/blob/master/docs/GLOSSARY.md

File: apache/license/贡献者许可协议常见问题.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+本页回答了我们收到的关于我们的贡献协议的大多数常见问题。
+
+* 对于其他授权问题，请参见我们的[授权常见问题]( https://www.apache.org/foundation/licensing.html)。
```
[the remark this answers is not in the document]
Comment (JesseZhou-1) [reply apache/license/贡献者许可协议常见问题.md:3]: 好的，我再推敲一下

**Review by WillemJiang — APPROVED** [review]

**Comment by WillemJiang** [comment]
@JesseZhou-1  因为之前你删除了文件， 导致合并的时候出现冲突。 
为了解决这个问题，你需要重新rebase一下 alc-beijing:master, 然后提交这个贡献者许可常见问题。
对了，建议使用英文文件名， 中文处理的会有些问题。

**Comment by JesseZhou-1** [comment]
好的

**Comment by JesseZhou-1** [comment]
那我是需要把这个PR关掉，然后重新提交一份PR么？

**Comment by WillemJiang** [comment]
> 那我是需要把这个PR关掉，然后重新提交一份PR么？

是的。

**Pull request closed without merging**